shared/js/background.js

```diff
@@ -32,8 +35,6 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
 });
 
 function Background() {
-  $this = this;
```

Comment:
For some reason, now that we're using Browserify, undefined variables cause an error where they didn't previously. (Maybe there's a `"use strict"` being now?)

`$this` is not defined nor used anywhere, so I removed it.